qemu/tests/seabios.py

```diff
@@ -60,13 +141,13 @@ def info_check():
         if not utils_misc.wait_for(info_check, timeout, 1):
             err_msg = "Cound not get sgabios message. The output"
             err_msg += " is %s" % get_output(vm.serial_console)
-            raise error.TestFail(err_msg)
+            test.error(err_msg)
 
     if restart_key:
-        error.context("Restart vm and check it's ok", logging.info)
+        logging.info("Restart vm and check it's ok")
```

Comment:
error.context is used to mention it's a step description, you can update to error_context.context

Reply:
Currently, @error.context_aware isn't recommended, But error.context depend on @error.context_aware.